src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -190,5 +183,13 @@ public new IOpenIdConnectEvents Events
         /// This is disabled by default.
         /// </summary>
         public bool UseTokenLifetime { get; set; }
+
+        /// <summary>
+        /// Indicates if requests to the CallbackPath may also be for other components. If enabled the middleware will pass
+        /// requests through that do not contain OpenIdConnect authentication responses. Disabling this and setting the
+        /// CallbackPath to a dedicated endpoint may provide better error handling.
+        /// This is enabled by default
+        /// </summary>
+        public bool ShareCallbackPath { get; set; } = true;
```

Comment:
Ignore/SkipUnrecognizedRequests would be what I understood the functionality actually does. Not sure if that's better though